files-csharp/DifferentWaysToOverwiteAFile/DifferentWaysToOverwiteAFile.Test/OverwriteExistingFileUnitTests.cs

```diff
@@ -0,0 +1,147 @@
+using DifferentWaysToOverwiteAFile.Examples;
+using System.Text;
+
+namespace DifferentWaysToOverwiteAFile.Test
+{
+    public class OverwriteExistingFileTests
+    {
+
+        [Test]
+        public void UseWriteAllText_ShouldOverwriteTheFile()
+        {
+            var filePath = "content.txt";
```

Comment:
Since this is repeated in all of the tests, we should promote it to a class constant